docs/tutorials/query-language.md

```diff
@@ -98,18 +109,46 @@ let teens = realm.objects('Contact').filtered('SUBQUERY(friends, $friend, $frien
 
 ### Backlink queries
 
-Other objects can link to an object and you can query on that releationship using the `@links` and `@links.ClassName.PropertyName` syntax.
-If the relationship of the LinkingObject is named, use the name in the query just like you would use any other property for readability.
-If the relationship is not named, you can use the `@links.ClassName.PropertyName` syntax where `ClassName.PropertyName` describes the forward relationship.
+Since backlinks are an indirect concept, let's consider a running example with the following models:
 
-Example:
 ```JS
-// Find contacts where someone from SF has them as friends
-realm.objects('Contact').filtered('@links.Contact.friends.city == "SF"');
+const CarSchema = {
+  name: 'Car',
+  properties: {
+    make:  'string',
+    model: 'string',
+    price: 'double',
+    owner: 'Person',
+  }
+};
+const PersonSchema = {
+  name: 'Person',
+  properties: {
+    name: 'string',
+    cars: { type: 'linkingObjects', objectType: 'Car', property: 'owner' }, // backlinks of Car.owner, this property is optional
+  }
+};
+```
 
-// Find contacts with no incomming links (across all linked properties)
-let isolated = realm.objects('Contact').filtered('@links.@count == 0');
+Links are properties which are an object type, for example `Car.owner` is a forward link. You can query through a forward link chain by following the link property names (see example 1 below). The query is self descriptive.
+What if we want to know which People own a certain type of car? We can find this using backlinks. Backlink is a term used to describe following a relationship backwards. This backwards relationship always exists for every link property but it is not required to name it in the model. We can use the named backlink (example 2 below) just like we did with a forward relationship. However, you can also query backlinks without a name by fully describing the forward relationship using the ` @links.ClassName.PropertyName` syntax (example 3 below). Regardless of whether they are named or not, backlinks can be treated like a collection. This means that you can use all the collection operators as usual (examples 4-7 below). In addition to the collection operators, there is special syntax to allow querying the count of all backlink relationships (example 8 below) which is different than querying for the backlink count on a specific relationship (example 5 below) unless there is only one incoming link in total. Although there is no functional difference between using named versus unnamed backlinks, the readability of the query syntax is affected so it is normally preferred to name the backlink with a `linkingObject` property when possible.
```

Comment:
Maybe: 
"property but it is not required" -> "property and it is not required"

Reply:
You mention "example 5" (and others), and cal it "Query 5" in the example. Maybe it should be "query 5 in the example below".